Article to create your First Dev Container Project

vikashsprem opened the pull request:
/claim #64
/closed #64

## Writer's Checklist

### Writing Structure
- [x] Use short sentences and paragraphs, and include bucket brigades.
- [x] Include more than two descriptive H2 headings to organize content.
- [x] Capitalize headings according to the AP Stylebook ([use this tool](https://capitalizemytitle.com/style/AP/))
- [x] Include an introduction with at least two paragraphs before the first H2 section.
- [x] Use appropriate formatting (bold, italic, underline), notes, quotes, TLDRs, and key points.
- [x] Incorporate `code elements` and `Markdown format` where appropriate.
- [x] Ensure at least one visual element per “screen” of the article (screenshots, diagrams, tables, graphs, lists, sidenotes, blockquotes).

### Fact-Checking
- [x] Verify all facts and data points included in the article.

### Asset Management
- [x] Save images in the `/assets` folder.
- [x] Follow naming conventions: `YYYYMMDD_title_of_the_article_IMG_NAME_NO.png`.
- [ ] (Optional) Create a GitHub repo for the code referenced in the article and share it.
- [ ] (Optional) Include a link to this Loom video in the PR comments.

### Interlinking
- [ ] Use `CTRL+F` to search for relevant keywords in:
    - [Daytona Dotfiles Sitemap](https://www.daytona.io/sitemap-dotfiles.xml)
    - [Daytona Definitions Sitemap](https://www.daytona.io/sitemap-definitions.xml)
- [x] Interlink keywords using the appropriate URLs.

### Glossary/Definitions
- [ ] Create new definition in `/defitnitions` folder.

### Review and Edit
- [x] Ensure articles address the needs of the target audience and their search intent.
- [x] Read the article out loud to catch any awkward phrasing.
- [x] Run the draft through [Grammarly](https://grammarly.com) or a similar grammar tool.
- [x] Double-check adherence to the style guide and repository guidelines.
- [x] Use the name of the article for the title of the PR.

vikashsprem commented:
@nkkko Please review the PR.

ntekim commented:
Is this still open or closed? I mean a PR was submitted but I still see the open tag.

mojafa commented:
@vikashsprem thank you for your patience.

Here are the changes you should make:

- Please fill out the author section and add image assets. you could refer to the [CONTRIBUTING.md](https://github.com/daytonaio/content/blob/main/CONTRIBUTING.md)

- See below, your artilce lacks consistent title and subtitle headings.
<img width="1512" alt="Screenshot 2024-11-29 at 07 29 54" src="https://github.com/user-attachments/assets/caa40bb2-4121-4d4b-99fa-c964d4a75ba6">
<img width="1512" alt="Screenshot 2024-11-29 at 07 30 19" src="https://github.com/user-attachments/assets/c4657a8a-6c90-47e2-b74e-b7edc8566580">
![Uploading Screenshot 2024-11-29 at 07.30.19.png…]()
<img width="1490" alt="Screenshot 2024-11-29 at 07 35 09" src="https://github.com/user-attachments/assets/c3828897-aefd-4e74-b369-41a44f190083">

vikashsprem commented:
Thank you @mojafa for letting me. I will resolve and let you know.

mojafa commented:
@vikashsprem any updates?

vikashsprem commented:
Hey @mojafa , Today i will push the latest commit.

mojafa commented:
Please take time and address the requested changes.

> See below, your artilce lacks consistent title and subtitle headings.

<img width="1256" alt="Screenshot 2024-12-05 at 02 46 21" src="https://github.com/user-attachments/assets/690305e3-3176-470f-a3f5-05a7a31b3b36">

**Please share screenshots of how these commands work**
<img width="1220" alt="Screenshot 2024-12-05 at 02 48 07" src="https://github.com/user-attachments/assets/10e35b38-e881-4c45-930d-1bcd825e38a6">

i dont think we have this command `daytona create test -r <repo> --skip-ide`

vikashsprem commented:
> **1 pending reviewer **

Hey @mojafa, I have fixed the title and subtitle headings consistency and added a shot to create the workspace, below step 3 have a reference so that you can verify the command. I might be looking at the previous commit (Check out the latest commit I made in which I addressed all the suggestions). Thanks

Daytonaio workspace create ref: https://www.daytona.io/docs/usage/workspaces/

vikashsprem commented:
> Please take time and address the requested changes.
> 
> > See below, your artilce lacks consistent title and subtitle headings.
> 
> <img alt="Screenshot 2024-12-05 at 02 46 21" width="1256" src="https://private-user-images.githubusercontent.com/38430072/392602753-690305e3-3176-470f-a3f5-05a7a31b3b36.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMzNzUwODIsIm5iZiI6MTczMzM3NDc4MiwicGF0aCI6Ii8zODQzMDA3Mi8zOTI2MDI3NTMtNjkwMzA1ZTMtMzE3Ni00NzBmLWEzZjUtMDVhN2EzMWIzYjM2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA1VDA0NTk0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTliZmI0NDk4YmZiZjBjMWY0YzJmZjMwNjcyZTdhMTgxMDQ5YTRlZTZmOGE1YTU0MjUyNTYzNzcyNDFhMzdmOWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.1ggiyPh4eEerJXiUfzP0RVVP2X6VbWa9DkvTxC2PG2U">
> **Please share screenshots of how these commands work** <img alt="Screenshot 2024-12-05 at 02 48 07" width="1220" src="https://private-user-images.githubusercontent.com/38430072/392602929-10e35b38-e881-4c45-930d-1bcd825e38a6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMzNzUwODIsIm5iZiI6MTczMzM3NDc4MiwicGF0aCI6Ii8zODQzMDA3Mi8zOTI2MDI5MjktMTBlMzViMzgtZTg4MS00YzQ1LTkzMGQtMWJjZDgyNWUzOGE2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA1VDA0NTk0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEwNmYzMjVhOTM0OWIzYzJiZGVlMDI1NTFkNWRiNjYzOWNhNmM2MDk4ZWIyZWFlNmJiZWEzNGMzZjgwZGM2MjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.XhrZFPOBhmcTPlUWqkYimfjtchlZVpP6CERRoELt0Rc">
> 
> i dont think we have this command `daytona create test -r <repo> --skip-ide`

There is no `Step 3a:` in the latest changes.

mojafa commented:
Looking into it

mojafa commented:
I'm still seeing the same stuff.
<img width="593" alt="Screenshot 2024-12-07 at 01 25 23" src="https://github.com/user-attachments/assets/8a1701e9-b5fc-4191-9750-67e011b52a1a">

mojafa commented:
See below.

Please also share some screenshots of how the codes/commands work.
<img width="713" alt="Screenshot 2024-12-07 at 01 30 12" src="https://github.com/user-attachments/assets/d0285e60-5aa9-4985-849c-173d90c57897">

vikashsprem commented:
Hey @mojafa, Now I have fixed the command. Plz look into it, Let me know if there is any issue?

mojafa commented:
we're almost good to go

mojafa commented:
@vikashsprem any updates

mojafa commented:
@vikashsprem  Thanks for your contribution and discussion on this issue! We're doing some cleanup to keep the repo focused, and we've closed the issue this PR addresses for now. Appreciate your support! 🙏
